java/src/jmri/jmrit/display/layoutEditor/LayoutEditorBundle.properties

```diff
@@ -267,6 +267,11 @@ ToolBarSideBottom = Bottom
 ToolBarSideRight = Right
 ToolBarSideFloat = Float
 ToolBarWide = Wide ToolBar
+ToolBox = Tool Box
```

Comment:
This should be "Tool Box: {0}" to allow translations where english grammar and the use of : is incorrect instead of translating just this word and building the string outside the localization.